src/entities/render-paywall-params.ts

```diff
@@ -0,0 +1,27 @@
+import { Offering } from "./offerings";
+
+/**
+ * Parameters for the {@link Purchases.renderPaywall} method.
+ * @public
+ */
+export interface RenderPaywallParams {
+  /**
+   * The identifier of the offering to fetch the paywall for.
+   * Can be a string identifier or one of the predefined keywords.
+   */
+  readonly offering: Offering;
```

Comment:
We could make this optional as well (so the whole params could be optional) and just use the `current` offering by default. Can be done in future PRs though 👍